<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>220</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>First formula</string>
     </attribute>
     <widget class="QLabel" name="X_label">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>X = </string>
      </property>
     </widget>
     <widget class="QLabel" name="Y_label">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Y = </string>
      </property>
     </widget>
     <widget class="QLabel" name="X_value">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>75</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="Y_value">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>115</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="Z_value">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>155</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>133</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="text">
          <string>a =</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="a_value"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>b=</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="b_value"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>183</width>
        <height>122</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../f_formula.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Посчитать</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="Z_label">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Z = </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Second formula</string>
     </attribute>
     <widget class="QLabel" name="Y_label_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Y =</string>
      </property>
     </widget>
     <widget class="QLabel" name="Y_value_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>65</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>136</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>x = </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="x_value"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>150</width>
        <height>94</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../s_formula.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Посчитать</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Third formula</string>
     </attribute>
     <widget class="QLabel" name="S_label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>S =</string>
      </property>
     </widget>
     <widget class="QLabel" name="S_value">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>75</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>141</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>m = </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_value"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>137</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>X = </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="X_value_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>144</width>
        <height>86</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../t_formula.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Посчитать</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>344</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
